Commons/Commons.vcxproj

```diff
@@ -25,6 +25,12 @@
       <PrecompiledHeader Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">NotUsing</PrecompiledHeader>
       <PrecompiledHeader Condition="'$(Configuration)|$(Platform)'=='Release|x64'">NotUsing</PrecompiledHeader>
     </ClCompile>
+    <ClCompile Include="commons\Guid.cpp">
```

Comment:
This looks very much like a html header, but I bet it's another language

Reply:
yep, it's a vcxproj visual studio c++ project file, which lays out all files, how they should be compiled etc. And like html, it's xml